docs/cross_toml.md

```diff
@@ -21,6 +21,7 @@ For example:
 
 ```toml
 [build.env]
+ignore-cargo-config = false
```

Comment:
We now use the `Cross.toml` config settings to allow people to skip the cargo config either via envvars or the config file.

Reply:
this is good, makes it more discoverable